### ISSUE TITLE
Create Telegraf role from existing configs

### ISSUE THREAD

**Issue by jwflory** [jwflory/infrastructure#24]
I need to add a Telegraf role and bring my existing configurations into sync. Ideally, it is a standalone role and when added to any of the existing host-based playbooks, it should "just work".

Blocked by #23.

**Comment by jwflory** [comment]
I'm no longer using InfluxDB or Telegraf, so closing this as `wontfix`.
